lms/views/api/sync.py

```diff
@@ -0,0 +1,43 @@
+import hashlib
+
+from pyramid.view import view_config
+
+from lms.models import HGroup, h_group_name
+
+
+@view_config(
+    route_name="api.sync", request_method="POST", renderer="json", permission="sync_api"
+)
+def sync(request):
+    authority = request.registry.settings["h_authority"]
+    canvas_api_svc = request.find_service(name="canvas_api_client")
+    lti_h_svc = request.find_service(name="lti_h")
+
+    context_id = request.json["course"]["context_id"]
+    custom_canvas_course_id = request.json["course"]["custom_canvas_course_id"]
```

Comment:
With this guy, was "custom_" added to distinguish it from standard LTI parameters, or does Canvas actually send us a request with "custom_canvas_course_id"?

I was just asking, because I wonder:

* If it's for transport only whether "canvas_course_id" would be the better name in the code
* ... or it's better to have the two match, even if it doesn't read as nice?